src/test/resources/security/sample.pem

```diff
@@ -1,28 +1,25 @@
 -----BEGIN CERTIFICATE-----
-MIIEyTCCA7GgAwIBAgIGAWLrc1O2MA0GCSqGSIb3DQEBCwUAMIGPMRMwEQYKCZIm
-iZPyLGQBGRYDY29tMRcwFQYKCZImiZPyLGQBGRYHZXhhbXBsZTEZMBcGA1UECgwQ
-RXhhbXBsZSBDb20gSW5jLjEhMB8GA1UECwwYRXhhbXBsZSBDb20gSW5jLiBSb290
-IENBMSEwHwYDVQQDDBhFeGFtcGxlIENvbSBJbmMuIFJvb3QgQ0EwHhcNMTgwNDIy
-MDM0MzQ3WhcNMjgwNDE5MDM0MzQ3WjBeMRIwEAYKCZImiZPyLGQBGRYCZGUxDTAL
-BgNVBAcMBHRlc3QxDTALBgNVBAoMBG5vZGUxDTALBgNVBAsMBG5vZGUxGzAZBgNV
-BAMMEm5vZGUtMC5leGFtcGxlLmNvbTCCASIwDQYJKoZIhvcNAQEBBQADggEPADCC
-AQoCggEBAJa+f476vLB+AwK53biYByUwN+40D8jMIovGXm6wgT8+9Sbs899dDXgt
-9CE1Beo65oP1+JUz4c7UHMrCY3ePiDt4cidHVzEQ2g0YoVrQWv0RedS/yx/DKhs8
-Pw1O715oftP53p/2ijD5DifFv1eKfkhFH+lwny/vMSNxellpl6NxJTiJVnQ9HYOL
-gf2t971ITJHnAuuxUF48HcuNovW4rhtkXef8kaAN7cE3LU+A9T474ULNCKkEFPIl
-ZAKN3iJNFdVsxrTU+CUBHzk73Do1cCkEvJZ0ZFjp0Z3y8wLY/gqWGfGVyA9l2CUq
-eIZNf55PNPtGzOrvvONiui48vBKH1LsCAwEAAaOCAVkwggFVMIG8BgNVHSMEgbQw
-gbGAFJI1DOAPHitF9k0583tfouYSl0BzoYGVpIGSMIGPMRMwEQYKCZImiZPyLGQB
-GRYDY29tMRcwFQYKCZImiZPyLGQBGRYHZXhhbXBsZTEZMBcGA1UECgwQRXhhbXBs
-ZSBDb20gSW5jLjEhMB8GA1UECwwYRXhhbXBsZSBDb20gSW5jLiBSb290IENBMSEw
-HwYDVQQDDBhFeGFtcGxlIENvbSBJbmMuIFJvb3QgQ0GCAQEwHQYDVR0OBBYEFKyv
-78ZmFjVKM9g7pMConYH7FVBHMAwGA1UdEwEB/wQCMAAwDgYDVR0PAQH/BAQDAgXg
-MCAGA1UdJQEB/wQWMBQGCCsGAQUFBwMBBggrBgEFBQcDAjA1BgNVHREELjAsiAUq
-AwQFBYISbm9kZS0wLmV4YW1wbGUuY29tgglsb2NhbGhvc3SHBH8AAAEwDQYJKoZI
-hvcNAQELBQADggEBAIOKuyXsFfGv1hI/Lkpd/73QNqjqJdxQclX57GOMWNbOM5H0
-5/9AOIZ5JQsWULNKN77aHjLRr4owq2jGbpc/Z6kAd+eiatkcpnbtbGrhKpOtoEZy
-8KuslwkeixpzLDNISSbkeLpXz4xJI1ETMN/VG8ZZP1bjzlHziHHDu0JNZ6TnNzKr
-XzCGMCohFfem8vnKNnKUneMQMvXd3rzUaAgvtf7Hc2LTBlf4fZzZF1EkwdSXhaMA
-1lkfHiqOBxtgeDLxCHESZ2fqgVqsWX+t3qHQfivcPW6txtDyrFPRdJOGhiMGzT/t
-e/9kkAtQRgpTb3skYdIOOUOV0WGQ60kJlFhAzIs=
```

Comment:
Certificates are updated in this PR: https://github.com/opensearch-project/anomaly-detection/pull/1020

Reply:
Seems like security tests are passing in #1020

Reply:
Look at the output here from the latest run on main: https://github.com/opensearch-project/anomaly-detection/actions/runs/6103610427/job/16564313234. Expand on `Pull and Run Docker`.

The output shows:

```
plugin version plugin_version qualifier docker_version
(opensearch-time-series-analytics-3.0.0.0-SNAPSHOT.zip) (analytics) (analytics) (3) (analytics)
```

which is showing the wrong values.

Reply:
Aah.. I see. This should be updated then.

The integTest doesn't pick up the demo certs. They are picked up by integTestRemote and since that is not run, this issue probly didn't surface early

Reply:
This PR still needs some updates. The backticks are fine, but the logic in the piped commands is not robustly handling the case where `plugin=opensearch-time-series-analytics-3.0.0.0-SNAPSHOT.zip`

```
plugin version plugin_version qualifier docker_version
(opensearch-time-series-analytics-3.0.0.0-SNAPSHOT.zip) (analytics) (analytics) (3) (analytics)
```

Reply:
So security tests are being skipped on main, but were being run on 2.x. The cert issue should have been spotted on CI in the 2.x branch within the last week after security merged the change to update the demo certs.

Reply:
@owaiskazi19 Was `anomaly-detection` renamed to `time-series-analytics`? If so, the cut commands in the `test-security` workflow on main need to be updated to account for the extra hyphen.

Instead of:

```
version=`echo $plugin|awk -F- '{print $4}'| cut -d. -f 1-3`
plugin_version=`echo $plugin|awk -F- '{print $4}'| cut -d. -f 1-4`
qualifier=`echo $plugin|awk -F- '{print $5}'| cut -d. -f 1-1`
```

it should become:

```
version=`echo $plugin|awk -F- '{print $5}'| cut -d. -f 1-3`
plugin_version=`echo $plugin|awk -F- '{print $5}'| cut -d. -f 1-4`
qualifier=`echo $plugin|awk -F- '{print $6}'| cut -d. -f 1-1`
```
on main, or find a more robust way of getting the values that can be run on both branches.

Reply:
Opened a PR to address the issue on main